### ISSUE TITLE
How is the complexity of training process?

### ISSUE THREAD

**Issue by wonlee2019** [LabShuHangGU/CCA#2]
![image](https://github.com/user-attachments/assets/304dea23-3346-4fec-bd72-5ede67db5649)
Notably, the training dataset has 300K images, while single 4090 is used. 
I want to know the degree of time consuming.

**Comment by MichTheEncoder** [comment]
When training with batch size of 8, it spends approximate 10 days to train the network. BTW, you can shorten the training time by decaying the learning rate early.

**Comment by wonlee2019** [comment]
Thanks a lot!
